docs/2fa/developers/workflow.md

```diff
@@ -0,0 +1,27 @@
+# Workflow
+
+![](../../assets/2fa/workflow.png)
+
+### Key Components
+* Primary wallet:
+
+  The place where users store and control their assets.
+
+* Recovery wallet:
+
+  Used for identifying the users' roles and resetting the 2FA device, the relationship between the primary wallet and the recovery wallet is saved on-chain after binding.
```

Comment:
Used for identifying users' roles and resetting the 2FA device. The relationship between the primary wallet and the recovery wallet is saved on-chain after binding.